telegram/_message.py

```diff
@@ -601,12 +641,12 @@ def __init__(
         self.edit_date = edit_date
         self.has_protected_content = has_protected_content
         self.text = text
-        self.entities = entities or []
-        self.caption_entities = caption_entities or []
+        self.entities = tuple(entities) if entities else ()
+        self.caption_entities = tuple(caption_entities) if caption_entities else ()
```

Comment:
Is it OK that an empty tuple is used here instead of `None`, like in most other cases? This it not the only place where I have the same question

Reply:
Yes, we have some attributes that where want to guarantee that you can iterate over them. See #2606 for some discussion

Reply:
See also a [later comment](https://github.com/python-telegram-bot/python-telegram-bot/pull/3249#discussion_r1035291164).